packages/edit-site/src/components/secondary-sidebar/list-view-sidebar.js

```diff
@@ -59,7 +60,7 @@ export default function ListViewSidebar() {
 					focusOnMountRef,
 				] ) }
 			>
-				<ListView />
+				<PrivateListView showAppender={ true } />
```

Comment:
This does make it active in the site editor, and I can't see any past conversations about that in the issue.

Reply:
Oh yep, there's no other way to test it 😄